opencti-platform/opencti-front/src/private/components/arsenal/Malwares.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import Grid from '@mui/material/Grid';
+import {
+  MalwaresCardsPaginationQuery$variables, MalwaresCardsPaginationQuery,
+} from './malwares/__generated__/MalwaresCardsPaginationQuery.graphql';
+import ListCards from '../../../components/list_cards/ListCards';
+import MalwaresCards, { malwaresCardsQuery } from './malwares/MalwaresCards';
+import MalwareCreation from './malwares/MalwareCreation';
+import Security from '../../../utils/Security';
+import { KNOWLEDGE_KNUPDATE } from '../../../utils/hooks/useGranted';
+import { usePaginationLocalStorage } from '../../../utils/hooks/useLocalStorage';
+import { Filters } from '../../../components/list_lines';
+import useQueryLoading from '../../../utils/hooks/useQueryLoading';
+import { GenericAttackCardDummy } from '../common/cards/GenericAttackCard';
+
+const LOCAL_STORAGE_KEY = 'view-malwares';
+
+const Malwares = () => {
+  const { viewStorage, helpers, paginationOptions } = usePaginationLocalStorage<MalwaresCardsPaginationQuery$variables>(
+    LOCAL_STORAGE_KEY,
+    {
+      searchTerm: '',
+      sortBy: 'name',
+      orderAsc: true,
+      openExports: false,
+      filters: {} as Filters,
+    },
+  );
+  const {
+    sortBy,
+    orderAsc,
+    searchTerm,
+    filters,
+    openExports,
+    numberOfElements,
+  } = viewStorage;
+
+  const queryRef = useQueryLoading<MalwaresCardsPaginationQuery>(
+    malwaresCardsQuery,
+    paginationOptions,
+  );
+
+  const renderCards = () => {
+    const dataColumns = {
+      name: {
+        label: 'Name',
+      },
+      created: {
+        label: 'Creation date',
+      },
+      modified: {
+        label: 'Modification date',
+      },
+    };
+    return (
+      <ListCards
+        sortBy={sortBy}
+        orderAsc={orderAsc}
+        dataColumns={dataColumns}
+        handleSort={helpers.handleSort}
+        handleSearch={helpers.handleSearch}
+        handleAddFilter={helpers.handleAddFilter}
+        handleRemoveFilter={helpers.handleRemoveFilter}
+        handleToggleExports={helpers.handleToggleExports}
+        openExports={openExports}
+        exportEntityType="Malware"
+        keyword={searchTerm}
+        filters={filters}
+        paginationOptions={paginationOptions}
+        numberOfElements={numberOfElements}
+        availableFilterKeys={[
+          'x_opencti_workflow_id',
+          'labelledBy',
+          'markedBy',
+          'createdBy',
+          'source_reliability',
+          'confidence',
+          'created_start_date',
+          'created_end_date',
+          'malware_types',
+          'revoked',
```

Comment:
Not relative to your work, but this filter is not working (not present in enum back-end)